packages/rspack-test-tools/tests/configCases/library/cjs-static/rspack.config.js

```diff
@@ -0,0 +1,7 @@
+/** @type {import("@rspack/core").Configuration} */
+module.exports = {
+	target: "node",
+	output: {
+		library: { type: "commonjs-static" }
```

Comment:
is the name 'commonjs-static' conventional ?

Reply:
Oh, I found it https://webpack.js.org/configuration/output/#type-commonjs-static